generic/ltp.py

```diff
@@ -112,18 +120,74 @@ def setUp(self):
                 "ltp-master%s" % match, locations=[url], expire='7d')
         else:
             self.cancel("Provided LTP Url is not valid")
-        archive.extract(tarball, self.workdir)
-        ltp_dir = os.path.join(self.workdir, "ltp-master")
+        self.ltpdir = '/home/ltp'
```

Comment:
any reason to use `/home` dir ? as using `tmp` space always be good

Reply:
having /tmp would be good

Reply:
@abdhaleegit @PraveenPenguin Done the changes as suggested